src/main/resources/hudson/plugins/synergy/SynergyFolderPublisher/config.jelly

```diff
@@ -1,11 +1,11 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-	<f:entry title="Copy folder only if the build is successful">
-		<f:checkbox name="synergyPublisher.publish" value="true" checked="${instance.isPublish()}"/>
+	<f:entry field="publish" title="Copy folder only if the build is successful">
+		<f:checkbox checked="${instance.isPublish()}" />

```

Comment:
If your form binding is done properly, there should be no need for a `checked` attribute.